PHPDoc comment in MessageOriginChat mislead to wrong getter getChat().

r3bers opened the pull request:
Factory creates getter getSender_chat() because property name sender_chat in MessageOriginChat

<!--
Important:
If this pull request is not related to any issue and contains a new feature, please create an issue first for discussion.
https://github.com/php-telegram-bot/core/issues/new?template=Feature_Request.md

Make sure this pull request is pointed towards the "develop" branch and refers to any issue that it's related to!
-->

<!-- Fill in the relevant information below to help triage your pull request. -->

| ?            |  !
|---           | ---
| Type         | bug
| BC Break     | yes / no
| Fixed issues |  New. 

#### Summary
 
Just simple quick fix because PHPDoc Comment was the same to MessageOriginChat and MessageOriginChannel but properties isn't

r3bers committed:
PHPDoc mislead to wrong getter getChat().
Factory creates getter getSender_chat() because property name sender_chat in MessageOriginChat

TiiFuchs commented:
It should be getSenderChat().
The snake case field names get converted to PascalCase. 

Could you adjust it accordingly, please?

r3bers committed:
PHPDoc mislead to wrong getter getChat().
Factory creates getter getSenderChat() because property name sender_chat in MessageOriginChat

r3bers commented:
Oh Sorry, my bad. Too much python for today. Pushed getSenderChat()

r3bers committed:
Always has cleanup errors in logs.
Because bot has channel functionality and cleanup of telegram_updates table didn't clean lines with links to tables messages and edited_messages by post columns.
